docs/dev/start-guide.md

```diff
@@ -1,37 +1,60 @@
 # Onboarding to Atlas Operator
-
-1. Install Go (1.16)
-2. Install dependencies
-```
-# on Mac
-brew install kubebuilder # or https://book.kubebuilder.io/quick-start.html
-brew install kustomize # or https://kubectl.docs.kubernetes.io/installation/kustomize/
-brew install act # or https://github.com/nektos/act#installation
-brew install yq # or https://pypi.org/project/yq/
-brew install shellcheck # or https://github.com/koalaman/shellcheck#installing
-brew install coreutils # or https://www.gnu.org/software/coreutils/
-brew install pre-commit # or https://pre-commit.com/index.html#install
-pre-commit install # from the root of the project
-```
-3. Install Kind ([instructions](https://kind.sigs.k8s.io/docs/user/quick-start/#installation))
-4. Clone the project to your workspace (note, that this doesn't need to be `GOPATH` as the project uses Go Modules)
-5. Copy the default Github Actions settings for local run: `cp .actrc.local.sample .actrc`
-6. Copy the default Github Actions environment for local run: `cp dotenv.sample .env`
-7. Update the .actrc - specify your Atlas connectivity data (orgId, keys)
-8. Build and deploy the Operator into the K8s cluster: `make deploy`
-9. Create an AtlasProject: `kubectl apply -f config/samples/atlas_v1_atlasproject.yaml` (note, that Atlas connection secrets are
- created during running `make deploy`)
-10. Create an AtlasDeployment: `kubectl apply -f config/samples/atlas_v1_atlasdeployment.yaml`
-11. Create an AtlasDatabaseUser: `kubectl apply -f config/samples/atlas_v1_atlasdatabaseuser.yaml`
+1. Install devbox by following the instructions provided on the official [Jetify website](https://www.jetify.com/devbox).
+2. Run the Devbox shell (by entering ```devbox shell``` in the bash to initialize an environment with pre-installed dependencies). For more details, refer to the devbox.json file.
+3. Install optional dependencies
```

Comment:
Can many of these steps not be automated via devbox? Having this as close to a one-click 'install' would be awesome.

Reply:
We could, but I will probably do it as a follow up.

Probably kind can be added to devbox, so it does not need separate install. And we could run some scripts at entering devbox shell.